src/js/modules/articles/selectors.js

```diff
@@ -63,10 +74,10 @@ export const getArticleFromRequestedSlug = createSelector(
 );
 
 /**
- * The selector returns a filtered articles object that contains all articles
+ * The selector returns an object that contains all articles
  *   written by a contributor.
  */
-export const getArticlesByContributor = createSelector(
+export const getContributorArticles = createSelector(
```

Comment:
jk it wasn't changed. I'm not going to make it "requested" because then I'd have to change getIllustratorArticles and photo to getRequestedIllustratorArticles in the interest of uniformity, but that doesn't make sense for the selector because, even though it IS getting a requested illustrator, it is not directly getting it in articles/selectors. See the code.